File: vs2003_cd01/Program Files/Microsoft Visual Studio .NET 2003/SDK/v1.1/Samples/Tutorials/clr_fundamentals/samples/typemembers/fields/cs/Fields.cs
using System;


class Days {

   public const Int32 minutesPerHour = 60;
   public const Int32 hoursPerDay = 24;

   public readonly Int32 minutesPerDays;

   public Days(Int32 days) {
      minutesPerDays = days * minutesPerHour * hoursPerDay;
   }

   static void Main() {
      Console.WriteLine("Hours Per Day={0}, Minutes Per Hour={1}, Minutes Per Day={2}",
         hoursPerDay, minutesPerHour, hoursPerDay * minutesPerHour);

      Days days = new Days(5);

      Console.WriteLine("Number of minutes in 5 days={0}", days.minutesPerDays);

      Console.Write("Press Enter to close window...");
      Console.Read();
   }
}

File: vs2003_cd01/Program Files/Microsoft Visual Studio .NET 2003/SDK/v1.1/Samples/Tutorials/clr_fundamentals/samples/typemembers/fields/vb/Fields.vb
Imports System


Class Days

   Public Const minutesPerHour As Int32 = 60
   Public Const hoursPerDay As Int32 = 24

   Public ReadOnly minutesPerDays As Int32

   Public Sub New(days As Int32)
      minutesPerDays = days * minutesPerHour * hoursPerDay
   End Sub

   Shared Sub Main()
      Console.WriteLine("Hours Per Day={0}, Minutes Per Hour={1}, Minutes Per Day={2}", _
         hoursPerDay, minutesPerHour, hoursPerDay * minutesPerHour)

      Dim days As New Days(5)

      Console.WriteLine("Number of minutes in 5 days={0}", days.minutesPerDays)

      Console.Write("Press Enter to close window...")
      Console.Read()
   End Sub
End Class
